src/app/lib/views/browser/list.js

```diff
@@ -237,25 +233,36 @@
         },
 
         itemsPerRow: function () {
-            var total = 0;
-            var items = $(document.querySelector('.items')).children('.item');
+            var currentWidth = this.$el.width();
+            var minPerRow = Math.floor(1 / (Settings.postersWidth / currentWidth) - 1);
+
+            // recalc only if needed
+            if (
+                currentWidth !== this.ui.row.listWidth // win was resized?
+                || Settings.postersWidth !== this.ui.row.posterWidth // poster were resized?
+                || this.ui.row.itemsPerRow < minPerRow // something impossible happened?
```

Comment:
returning early here makes the code more readable and introduces less whitespace changes.